src/VirtualMonoRepo/source-mappings.json

```diff
@@ -54,6 +54,61 @@
         {
             "name": "arcade",
             "defaultRemote": "https://github.com/dotnet/arcade",
+            "include": [
+                "src/Microsoft.DotNet.Build.Tasks.Feed.Tests/TestInputs/Nupkgs/test-package-a.1.0.0.nupkg",
```

Comment:
Why not exclude the whole `src/Microsoft.DotNet.Build.Tasks.Feed.Tests/` directory? It won't be possible to run these tests in the VMR with all these files deleted, so you can as well exclude the whole directory.

Reply:
dtto for src/Microsoft.DotNet.NuGetRepack and src/Microsoft.DotNet.SignTool.Tests